### DISCUSSION TITLE
Shortcut to open the context on the browser

### DISCUSSION THREAD

**Discussion by Drarig29** [community/community#3848] [Mobile] [answered]
Sometimes, a feature is missing in the Android app, and I'd like to tap on a new action bar button which sends me to the browser with the same context as I am right now (if possible).

For example, I'm on the homepage of a repo and the search feature is missing: I'll be sent to the homepage in the browser.

I'm in the issue #11, I'll be sent to this issue #11 on my browser.

Sometimes the web interface in desktop mode is more comfy... 😁

**Comment by nplasterer** [comment, marked as answer]
Hey @Drarig29 do you see the top right share icon? You should be able to click on the icon in the top right corner of most screens to take you to that screen on the web. Let me know if that makes sense and helps with your issue.

**Reply by Drarig29** [reply]
Yes I see it, and among all the available options, the only interesting one is to "Copy the link". I can't quickly open it in my browser, I have to manually open it and paste the link.

**Reply by nplasterer** [reply]
Thanks for the context! I've filed an issue with the team and we will look into some improvements here.
